9-Tools/configfile_generator/README.md

```diff
@@ -1 +1,164 @@
-TODO
+# Introduction
+This is a script with several functions for easily managing Kubernetes config files such as: 
+
+* Generating config files for multiple user within different namespaces with a specified access role, 
+	Either choose one of the role existed (admin,user,viewer) or provide a role yaml file 
+* Generating config files for several users within already existed namespaces(so we won't need to create a new namespace) 
+	Either choose one of the role existed (admin,user,viewer) or provide a role yaml file 
+* Merging config files for the same user to access to several different namespace in the same or different clusters.
+* Limit resource over N namespaces with a provided resource-quota.yaml file.
+
+
+# Important to know before reading more
+
+Q: How to set config files ?
+
+A: In bash terminal (Note that you have to cd into the same folder as the config file)
+The safest way is to replace directly your config file with the config file in $HOME/.kube/config with this new file(Do keep a copy of the old one first in this case)
+
+There is also this other way with export KUBECONFIG="configfilename", but the variable rather being updated or expired too quickly so you will get an anonymous authentication error as if the configfilename has been unloaded.
+
+Q: How to re-set the origin config files?
+
+A:In bash terminal, Assumingly you have not move aroung you .kube folder or using a entirely different system than linux-based, the default path for your original config file should always be at $HOME/.kube/config
+* export KUBECONFIG="$HOME/.kube/config"
+
+Q: How to switch namespace or cluster, also know as switching context ?
+
+A: In bash terminal 
+* kubectl config get-contexts 
+
+This will give you all available contexts for your config file to switch to. Note that only "those" contexts in your config file so if you want to access a cluster or a namespace not in your config file you have to acquire the config file containing that context(ask the guy who can give you the config file).
+
+* kubectl config set-context contextname
+
+contextname can be gotten from the previous command, check name column
+
+
+# Dependencies
+This requires the following to work. 
+
+* Gcloud
```

Comment:
Kubernetes could be deployed on other platforms, so gcloud should not be required.

Reply:
Removed Gcloud.